### PR TITLE
Add a tweet for metric_template video

### PR DESCRIPTION
**Pull request by Shreyanand**
This PR closes https://github.com/aicoe-aiops/ocp-ci-analysis/issues/296 by adding a tweet to promote the metric template Youtube video.

### CODE REVIEW

**Review by harshad16 — APPROVED** [review]
/lgtm
thanks :100:

File: tweets/2021-06/metric_template.tweet [hunk 1]
```diff
@@ -0,0 +1 @@
+Interested in monitoring and improving your CI process? Be sure to checkout https://youtu.be/ouxtHH2vOHg to write notebooks for Key Performance Indicators using #AI4CI metric template.
```
Comment (chauhankaranraj) [inline tweets/2021-06/metric_template.tweet:1]:
```suggestion
Interested in monitoring and improving your CI process? Check out https://youtu.be/ouxtHH2vOHg to learn how to write Jupyter notebooks using the #AI4CI metric template to calculate Key Performance Indicators for CI.
```

**Review by chauhankaranraj — CHANGES_REQUESTED** [review]
Small suggestion to make the tweet clearer, lmk what you think :)

**Review by oindrillac — APPROVED** [review]
/lgtm 🐦 
/approve

**Pull request merged**